tools/bridge/bridge/control_events.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC
+
+import attr
+
+
+class ControlEvent(ABC):
+    pass
+
```

Comment:
I would probably rename that to Event. The FetcherReacherHeadEvent could probably also be moved here, but this one should not end up on the control queue!